core/controllers/home_test.py

```diff
@@ -26,8 +26,8 @@ class HomePageTest(test_utils.GenericTestBase):
     def test_logged_out_homepage(self):
         """Test the logged-out version of the home page."""
         response = self.testapp.get('/')
-        self.assertEqual(response.status_int, 200)
-        response.mustcontain(
+        self.assertEqual(response.status_int, 302)
```

Comment:
Still need to assert correct response URL location. This is done elsewhere in other python tests where we do a 302; you might want to grep.